proto/frequenz/api/weather/weather.proto

```diff
@@ -148,6 +152,11 @@ message LocationForecast {
 // corresponding to a requested location.
 message GetHistoricalWeatherForecastResponse {
   repeated LocationForecast location_forecasts = 1;
+
+  // The pagination_info contains the number of 'total_items' in the response
+  // and the 'next_page_token' for the subsequent query of the request. It is
+  // omitted in the last response.
+  optional frequenz.api.common.v1.pagination.PaginationInfo pagination_info = 2;
```

Comment:
Same: We do not necessarily need to specify it as `optional`, because `PaginationParams` is optional by default.